### DISCUSSION TITLE
Is there a way to display definition of blade component inside <x-markdown>?

### DISCUSSION THREAD

**Discussion by lichtner** [spatie/laravel-markdown#16] [Ideas]
I have simple blade component `resources\views\components\hello`

    <h1>hello {{ $name }}</h1>

and I want to write documentation about this component, how to use it:

    <x-markdown>
    You can use `<x-hello>` component like this: 

    ```blade
    <x-hello name="world" />
    ```
    </x-markdown>

but on the page I see already compiled result. 

Is there a way to suppress compilation inside markdown?
If not, could you add something like this?

    <x-markdown suppress-compilation="true">

**Comment by freekmurze** [comment]
Feel free to PR this
